test/metabase/driver/postgres_test.clj

```diff
@@ -1697,3 +1698,57 @@
                                                                         nil))
                           (lib/limit 1))]
             (is (->> query qp/process-query mt/rows))))))))
+
+(deftest column-attributes-test
+  (mt/test-driver :postgres
+    (drop-if-exists-and-create-db! "generated_cols_test")
+    (let [details  (mt/dbdef->connection-details :postgres :db {:database-name "generated_cols_test"})
+          spec     (sql-jdbc.conn/connection-details->spec :postgres details)
+          describe
+          (fn [column-defs]
+            (mt/with-temp [:model/Database database {:engine :postgres, :details details}]
+              (jdbc/execute! spec ["DROP TABLE IF EXISTS test_table"])
```

Comment:
I'd feel safer with a random name

Reply:
+1, and also in `column-attributes-test`